cmd/tetragon/flags.go

```diff
@@ -94,7 +94,6 @@ func readAndSetFlags() {
 
 	option.Config.EnableProcessCred = viper.GetBool(keyEnableProcessCred)
 	option.Config.EnableProcessNs = viper.GetBool(keyEnableProcessNs)
-	option.Config.EnableCilium = viper.GetBool(keyEnableCiliumAPI)
```

Comment:
Would it make sense to keep the option and only fail it is set to true? I _think_ this would enable us to run the `:latest` image with the old helm charts that define this option. We can remove it once we have released the next versin.